Nodejs/Product/Nodejs/Repl/NpmReplCommand.cs

```diff
@@ -76,19 +76,9 @@ class NpmReplCommand : IReplCommand {
                     continue;
                 }
 
-                EnvDTE.Properties properties = dteProject.Properties;
-                if (dteProject.Properties == null) {
-                    continue;
-                }
-
                 string projectName = dteProject.Name;
-                EnvDTE.Property projectHome = properties.Item("ProjectHome");
-                if (projectHome == null || projectName == null) {
-                    continue;
-                }
-
-                var projectDirectory = projectHome.Value as string;
-                if (projectDirectory != null) {
+                string projectDirectory = Path.GetDirectoryName(dteProject.FullName);
```

Comment:
Here's an example of what `EnvDte.properties` contains for a asp app (only printing out string property values):

```
Count = 102
    [0]: {(WebApplication.AspNetDebugging, )}
    [1]: {(WebApplication.AspnetCompilerIISMetabasePath, )}
    [2]: {(OutputTypeEx, )}
    [3]: {(TargetFrameworkMoniker, .NETFramework,Version=v4.5.2)}
    [4]: {(ComVisible, )}
    [5]: {(EnableSecurityDebugging, )}
    [6]: {(OptionCompare, )}
    [7]: {(StartupObject, )}
    [8]: {(WebApplication.SSLEnabled, )}
    [9]: {(WebApplication.UseIIS, )}
    [10]: {(ManifestCertificateThumbprint, )}
    [11]: {(Trademark, )}
    [12]: {(Title, WebApplication1)}
    [13]: {(WebApplication.StartExternalUrl, )}
    [14]: {(WebApplication.IISUrl, http://localhost:63763/)}
    [15]: {(AssemblyOriginatorKeyFileType, )}
    [16]: {(FileName, WebApplication1.csproj)}
    [17]: {(WebServer, )}
    [18]: {(AssemblyOriginatorKeyMode, )}
    [19]: {(AssemblyKeyContainerName, )}
    [20]: {(WebApplication.WindowsAuthenticationEnabled, )}
    [21]: {(WebApplication.SecureUrl, )}
    [22]: {(WebApplication.DevelopmentServerCommandLine,  /config:"C:\Users\matb.REDMOND\Documents\Visual Studio 2015\Projects\WebApplication1\.vs\config\applicationhost.config" /site:"WebApplication1" /apppool:"Clr4IntegratedAppPool")}
    [23]: {(WebApplication.SQLDebugging, )}
    [24]: {(WebApplication.StartPageUrl, )}
    [25]: {(WebApplication.DefaultServerDirectoryListing, )}
    [26]: {(WebApplication.DevelopmentServerVPath, /)}
    [27]: {(WebApplication.DevelopmentServerPort, )}
    [28]: {(ProjectType, )}
    [29]: {(ReferencePath, )}
    [30]: {(WebApplication.IsUsingIISExpress, )}
    [31]: {(PreBuildEvent, )}
    [32]: {(WebApplication.AnonymousAuthenticationEnabled, )}
    [33]: {(WebApplication.SilverlightDebugging, )}
    [34]: {(WebApplication.StartCmdLineArguments, )}
    [35]: {(Copyright, Copyright ©  2016)}
    [36]: {(ApplicationIcon, )}
    [37]: {(WebApplication.CurrentDebugUrl, http://localhost:63763/)}
    [38]: {(ExcludedPermissions, )}
    [39]: {(RunPostBuildEvent, )}
    [40]: {(DefaultTargetSchema, )}
    [41]: {(RootNamespace, WebApplication1)}
    [42]: {(WebApplication.IsUsingCustomServer, )}
    [43]: {(ManifestTimestampUrl, )}
    [44]: {(ManifestKeyFile, )}
    [45]: {(DebugSecurityZoneURL, )}
    [46]: {(Product, WebApplication1)}
    [47]: {(PostBuildEvent, )}
    [48]: {(OptionStrict, )}
    [49]: {(DefaultHTMLPageLayout, )}
    [50]: {(DelaySign, )}
    [51]: {(OutputType, )}
    [52]: {(WebApplication.StartWorkingDirectory, )}
    [53]: {(WebApplication.DebugStartAction, )}
    [54]: {(NeutralResourcesLanguage, )}
    [55]: {(OptionExplicit, )}
    [56]: {(OutputFileName, WebApplication1.dll)}
    [57]: {(ServerExtensionsVersion, )}
    [58]: {(WebApplication.NonSecureUrl, http://localhost:63763/)}
    [59]: {(WebApplication.ToString, Microsoft.VisualStudio.Web.Application.WAProjectExtender)}
    [60]: {(AssemblyGuid, 295f02da-71a5-4ad9-a3b4-cfc9544dd82a)}
    [61]: {(GenerateManifests, )}
    [62]: {(AssemblyVersion, 1.0.0.0)}
    [63]: {(Win32ResourceFile, )}
    [64]: {(Description, )}
    [65]: {(URL, file:///C:\Users\matb.REDMOND\Documents\Visual Studio 2015\Projects\WebApplication1\WebApplication1\)}
    [66]: {(DefaultClientScript, )}
    [67]: {(WebApplication.NativeDebugging, )}
    [68]: {(TargetFramework, )}
    [69]: {(SignManifests, )}
    [70]: {(OfflineURL, )}
    [71]: {(WebServerVersion, )}
    [72]: {(Publish, )}
    [73]: {(AssemblyType, )}
    [74]: {(FullPath, C:\Users\matb.REDMOND\Documents\Visual Studio 2015\Projects\WebApplication1\WebApplication1\)}
    [75]: {(WebAccessMethod, )}
    [76]: {(WebApplication.UseIISExpress, )}
    [77]: {(WebApplication.BrowseURL, http://localhost:63763)}
    [78]: {(WebApplication.NTLMAuthentication, )}
    [79]: {(WebApplication.OverrideIISAppRootUrl, )}
    [80]: {(WebApplication.OpenedURL, file:///C:/Users/matb.REDMOND/Documents/Visual Studio 2015/Projects/WebApplication1/WebApplication1/)}
    [81]: {(AssemblyKeyProviderName, )}
    [82]: {(TypeComplianceDiagnostics, )}
    [83]: {(Company, )}
    [84]: {(ActiveFileSharePath, )}
    [85]: {(AssemblyOriginatorKeyFile, )}
    [86]: {(WebApplication.StartWebServerOnDebug, )}
    [87]: {(WebApplication.AutoAssignPort, )}
    [88]: {(ApplicationManifest, DefaultManifest)}
    [89]: {(AssemblyFileVersion, 1.0.0.0)}
    [90]: {(AspnetVersion, )}
    [91]: {(FileSharePath, )}
    [92]: {(AssemblyName, WebApplication1)}
    [93]: {(WebApplication.EditAndContinue, )}
    [94]: {(WebApplication.ServerDirectoryListing, )}
    [95]: {(LocalPath, C:\Users\matb.REDMOND\Documents\Visual Studio 2015\Projects\WebApplication1\WebApplication1\)}
    [96]: {(DefaultNamespace, WebApplication1)}
    [97]: {(LinkRepair, )}
    [98]: {(WebApplication.StartExternalProgram, )}
    [99]: {(WebApplication.IISAppRootUrl, )}
    [0]: {(TargetZone, )}
    [1]: {(SignAssembly, )}
```